custom_components/opnsense/coordinator.py

```
@@ -51,58 +52,44 @@
+            else:
+                _LOGGER.error(f"Method {cat.get('function','')} not found.")
+
+        results: list = await asyncio.gather(*tasks, return_exceptions=True)
+
+        for i, cat in enumerate(categories):
+            if not isinstance(results[i], Exception):
+                state[cat.get("state_key")] = results[i]
+            else:
+                _LOGGER.error(
+                    f"Error getting {cat.get('state_key')}. "
+                    f"{results[i].__class__.__qualname__}: {results[i]}"
```

Ruff

Logging statement uses f-string

(G004)